src/library/librarycontrol.cpp

```diff
@@ -337,14 +337,31 @@ LibraryControl::LibraryControl(Library* pLibrary)
     // Track Color controls
     m_pTrackColorPrev = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "track_color_prev"));
     m_pTrackColorNext = std::make_unique<ControlPushButton>(ConfigKey("[Library]", "track_color_next"));
+    m_pTrackColorSelector = std::make_unique<ControlEncoder>(
+            ConfigKey("[Library]", "track_color_selector"), false);
     connect(m_pTrackColorPrev.get(),
             &ControlPushButton::valueChanged,
             this,
-            &LibraryControl::slotTrackColorPrev);
+            [this](double value) {
+                if (value > 0) {
+                    LibraryControl::slotTrackColorSelector(-1);
+                }
```

Comment:
you're calling member functions of the same class. Just do this. Otherwise it looks like you're calling a parent or static member function. 
```suggestion
                if (value > 0) {
                    slotTrackColorSelector(-1);
                }
```